### DISCUSSION TITLE
Mod select map stats ordering is inconsistent

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#24834] [Q&A]
On the osu! website and in osu!lazer, map stats are ordered CS -> HP -> OD -> AR. I feel that it would make more sense if the map stats shown in mod select were ordered in the same way.

osu!lazer:
![image](https://github.com/ppy/osu/assets/74088315/48b4dc9f-1cc3-4220-a4ba-9ae360fe0860)

osu! website:
![image](https://github.com/ppy/osu/assets/74088315/0f17f20c-d9c6-4257-9e8a-1b468ca5ab79)

Mod select map stats:
![image](https://github.com/ppy/osu/assets/74088315/da94bfc3-d7e2-4648-bdbc-8325c63f38a6)

**Comment by peppy** [comment]
Meanwhile, on stable it's `CS AR OD HP`.

**Reply by Rekunan** [reply]
My personal favorite since CS and AR tell the most about the map compared to OD and HP.
